src/get-started/java-binding.md

```diff
@@ -286,6 +288,8 @@ rules – see the corresponding section of our [documentation][transactions].
 
 #### Messages
 
+<!-- todo: @ostrovski Shall we keep this overview, or just keep the link? -->
```

Comment:
As I've commented above, I'm for keeping a link + possibly a couple of sentences to provide necessary immediate context.

Reply:
Ok, thanks, I'll see how I could condense these two sections into some kind of an introduction (Messages + Lifecycle).

Reply:
Replaced with a sentence on messages and a link.

Reply:
497b617